packages/dd-trace/src/log.js

```diff
@@ -1,27 +1,22 @@
 'use strict'
 
 const _default = {
-  debug: message => console.log(message), /* eslint-disable-line no-console */
-  error: err => console.error(err) /* eslint-disable-line no-console */
+  debug: (...args) => console.log(...args), /* eslint-disable-line no-console */
+  info: (...args) => console.error(...args), /* eslint-disable-line no-console */
+  warn: (...args) => console.error(...args), /* eslint-disable-line no-console */
+  error: (...args) => console.error(...args) /* eslint-disable-line no-console */
```

Comment:
we could use more specific console methods:

```suggestion
  debug: console.debug, /* eslint-disable-line no-console */
  info: console.info, /* eslint-disable-line no-console */
  warn: console.warn, /* eslint-disable-line no-console */
  error: console.error /* eslint-disable-line no-console */
```

Reply:
I tried, but this breaks the tests, which stub console. Roch's suggesting of stubbing higher up at the logger would solve this though.